Tools/autotest/vehicle_test_suite.py

```diff
@@ -501,10 +506,39 @@ def timeoutexception(self):
         return AutoTestTimeoutException(f"Failed to get EKF.flags={self.required_flags}")
 
     def progress_text(self, current_value):
-        error_bits_str = ""
-        if current_value & self.error_bits:
-            error_bits_str = " (error bits present)"
-        return (f"Want=({self.required_flags}) got={current_value}{error_bits_str}")
+        error_bits = current_value & self.error_bits
+        return (f"Want={self.required_flags} got={current_value} errors={error_bits}")
+
+    def ekf_flags_string(self, bits):
+        ret = []
+        for i in range(0, 16):
+            bit = 1 << i
+            try:
+                if not bits & bit:
+                    continue
+                name = mavutil.mavlink.enums["ESTIMATOR_STATUS_FLAGS"][bit].name
+                trimmed_name = name.removeprefix("ESTIMATOR_")
+                ret.append(trimmed_name)
+            except KeyError:
+                ret.append(str(bit))
+        return "|".join(ret)
+
+    def failure_text(self, now, current_value):
```

Comment:
now ?

Reply:
> now ?

Yeah, just in case :-)  It mirrors what we pass through to "progress_text", and can be compared against initial values IIRC.